spark/src/main/java/io/ddf/spark/content/SchemaHandler.java

```diff
@@ -192,6 +199,7 @@ public static Schema getSchemaFromDataFrame(DataFrame rdd) {
         return row != null;
       });
       dataFrame = manager.getHiveContext().createDataFrame(appliedRdd.rdd(), newSchema);
+      dataFrame.cache();
```

Comment:
More importantly, if you move the `cache()` call to inside `applySchema` only, what will happen to other DDFs that created with `copyFrom` like with ORC or PQT files?